src/Autumn/compile.jl

```diff
@@ -4,11 +4,140 @@ module Compile
 using ..AExpressions
 using ..Program
 
+export compileToJulia, toRepr
+
+struct AutumnCompileError <: Exception
+  msg
+end
+
+fixedSymbols = [:+, :-, :/, :*, :&&, :||, :>=, :<=, :>, :<, :(==)]
+
+AutumnCompileError() = AutumnCompileError("")
+
 "Compile `aexpr` into `program::Program`"
-function compiletojulia(::AExpr)::AProgram
-  # Do type inference
-  # Get external values
-  # Alot
+function compileToJulia(aexpr::AExpr) #::AProgram
```

Comment:
Julia only uses CamelCase for type names by convention.  use `compiletojulia` or `compile_to_julia`